.circleci/config.yml

```diff
@@ -356,7 +384,7 @@ workflows:
               only: /v\d+\.\d+\.\d+(-[\w]+)?/
       - test:
           name: test-3.8
-          extras: '[unittest]'
+          extras: '[unittest,typecheck]'
```

Comment:
I took this as an opportunity to further pull things out of requirements.txt into appropriate extras - including mypy under the new typecheck extra.